netdisco/discoverables/logitech_mediaserver.py

```diff
@@ -11,4 +11,5 @@ def __init__(self, netdis):
 
     def get_entries(self):
         """Get all the Logitech Media Server details."""
-        return [entry['from'] for entry in self.netdis.lms.entries]
+        return [(entry['from'], entry['port'])
```

Comment:
I want to migrate all netdisco `get_entries` calls to return a dict. Can you make it just return `self.netdis.lms.entries` ?